From bb86821aaef6fab8f807eafd76761b3ed68f4de4 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Thu, 26 Jun 2014 14:24:42 -0400
Subject: [Documentation] MAINTAINERS: Add Tegra PCIe maintainer

Message-id: <20140626142442.18681.49016.stgit@gir.stowe>
Patchwork-id: 84964
O-Subject: [RHEL7.1 PATCH 1/7] MAINTAINERS: Add myself as Tegra PCIe maintainer
Bugzilla: 1110890
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110890
Upstream Status: 0447cfd75a6bad561ebf2b393dcb2263c261b6c5

commit 0447cfd75a6bad561ebf2b393dcb2263c261b6c5
Author: Thierry Reding <thierry.reding@gmail.com>
Date:   Fri Aug 9 16:49:32 2013 +0200

    MAINTAINERS: Add myself as Tegra PCIe maintainer

    I'll be taking on maintainership of the Tegra PCIe driver since it's now
    moved out of arch/arm/mach-tegra.

    Signed-off-by: Thierry Reding <treding@nvidia.com>
    Acked-by: Bjorn Helgaas <bhelgaas@google.com>
    Signed-off-by: Stephen Warren <swarren@nvidia.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/MAINTAINERS b/MAINTAINERS
index 5f77bd6..bba2ab7 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -6200,6 +6200,13 @@ F: drivers/pci/
 F: include/linux/pci*
 F: arch/x86/pci/
 
+PCI DRIVER FOR NVIDIA TEGRA
+M: Thierry Reding <thierry.reding@gmail.com>
+L: linux-tegra@vger.kernel.org
+S: Supported
+F: Documentation/devicetree/bindings/pci/nvidia,tegra20-pcie.txt
+F: drivers/pci/host/pci-tegra.c
+
 PCMCIA SUBSYSTEM
 P: Linux PCMCIA Team
 L: linux-pcmcia@lists.infradead.org
-- 
1.7.1